common/src/tflite.cc

```diff
@@ -171,6 +171,10 @@ void tflite_load_model(const unsigned char* model_data) {
     return;
   }
 
+#ifdef TF_LITE_SHOW_MEMORY_USE
+  interpreter->GetMicroAllocator().PrintAllocations();
+#endif
```

Comment:
PrintAllocations() provides the info from arena_used_bytes() and much more, correct?

Reply:
Yes, that's right.